src/framework/global/io/fileinfo.cpp

```diff
@@ -145,37 +145,25 @@ QDateTime FileInfo::lastModified() const
     return fileSystem()->lastModified(m_filePath);
 }
 
-#if defined(Q_OS_WIN)
 bool FileInfo::isRelative() const
 {
-    return m_filePath.isEmpty()
-           || (m_filePath.at(0) != '/'
-               && !(m_filePath.length() >= 2 && m_filePath.at(1) == ':'));
+    return !isAbsolute();
 }
```

Comment:
not sure if we need to support unc paths etc., then there's the "\\?\..." convention...are all pathnames in MuseScore passed around using forward slash?